spec/brainstem-model-spec.coffee

```diff
@@ -94,14 +139,48 @@ describe 'Brainstem.Model', ->
       expect(model.fetch()).toEqual(promise)
 
     describe 'integration', ->
-      it 'something', ->
-        task = buildTask()
-        respondWith(server, '/api/tasks/1', resultsFrom: 'tasks', data: task)
+      it 'fetches model', ->
+        updatedModel = buildTask()
+
+        respondWith(server, "/api/tasks/#{model.id}", resultsFrom: 'tasks', data: updatedModel)
 
         model.fetch()
         server.respond()
 
-        expect(model.attributes).toEqual(task.attributes)
+        expect(model.attributes).toEqual(updatedModel.attributes)
```

Comment:
I think the `id` would be different, but I added a differing description to be sure :) Good point.